### PR TITLE
generate docs for dart sdk

### PR DESCRIPTION
**Pull request by keertip**
@sethladd

### CODE REVIEW

File: lib/dartdoc.dart [hunk 1]
```diff
@@ -73,14 +82,13 @@ class DartDoc {
...
-
+  List<LibraryElement> _parseLibraries(List<String> files) {
     DartSdk sdk = new DirectoryBasedDartSdk(new JavaFile(_sdkDir.path));
-
     ContentCache contentCache = new ContentCache();
-    List<UriResolver> resolvers = [new DartUriResolver(sdk), new FileUriResolver()];
-    JavaFile packagesDir = new JavaFile.relative(new JavaFile(_rootDir.path), 'packages');
+    List<UriResolver> resolvers = [new DartUriResolver(sdk),
+                                   new FileUriResolver()];
+    JavaFile packagesDir =
```
Comment (sethladd) [inline lib/dartdoc.dart]: This always confuses me. When can we rename JavaFile? :)

File: test/model_test.dart [hunk 2]
```diff
@@ -67,6 +69,20 @@ tests() {
...
     });
+
+    var p2 = new Package([e], Directory.current.path, '1.9.0-dev.3.0', true);
+
+    test('sdk name', () {
+      expect(p2.name, 'Dart API Reference');
+    });
+
+    test('sdk version', () {
+      expect(p2.version, '1.9.0-dev.3.0');
```
Comment (sethladd) [inline test/model_test.dart]: will this change over time? will that make it hard to keep updated?

**Comment by sethladd** [comment]
Woohoo!

File: lib/dartdoc.dart (hunk 1 above)
Comment (keertip) [inline lib/dartdoc.dart]: Will talk to the analyzer folks.

File: test/model_test.dart (hunk 2 above)
Comment (keertip) [inline test/model_test.dart]: No, this is just a test and does not need to be updated.

File: lib/dartdoc.dart (hunk 1 above)
Comment (sethladd) [inline lib/dartdoc.dart]: https://code.google.com/p/dart/issues/detail?id=22075 :)

**Pull request merged**